src/components/TransactionDetails/CreatePairTransactionDetails.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <component :is="infoOnly ? 'div' : 'transaction-details'" class="info-line-container">
```

Comment:
I'm not sure that this is a good idea to use `info-line-container` class here, because you use it for inner blocks.

Reply:
It should be there as it breaks layout

<img width="422" alt="Screenshot 2021-12-23 at 18 00 15" src="https://user-images.githubusercontent.com/27702061/147257727-c138ea73-514c-47fd-824d-60b534d12940.png">